assets/js/vue/components/search/DatasetRow.vue

```diff
@@ -56,6 +58,24 @@ export default {
         window.open(`/data/${this.datasetRowData.udi}`, '_blank');
       }
     },
+    fileFormat() {
+      if (this.datasetRowData.datasetSubmission) {
+        return this.datasetRowData.fileFormat ?? this.datasetRowData.datasetSubmission.distributionFormatName;
+      }
+      return this.datasetRowData.fileFormat ?? null;
+    },
+    fileSize() {
+      if (this.datasetRowData.datasetSubmission) {
+        return this.datasetRowData.fileSize ?? xbytes(this.datasetRowData.datasetSubmission.datasetFileSize);
+      }
+      return this.datasetRowData.fileSize ?? null;
+    },
+    coldStorage() {
+      if (this.datasetRowData.datasetSubmission) {
+        return this.datasetRowData.coldStorage ?? this.datasetRowData.datasetSubmission.coldStorage;
+      }
+      return this.datasetRowData.coldStorage ?? false;
+    },
```

Comment:
It is a bit redundant/confusing, we should rewrite this into a simple if-else?

Reply:
This should go away in a future ticket.